solutions/experimentation/core-landing-zone/org/org-sink.yaml

```diff
@@ -62,10 +62,10 @@ spec:
 apiVersion: logging.cnrm.cloud.google.com/v1beta1
 kind: LoggingLogSink
 metadata:
-  name: logging-project-id-google-workspace-data-access-sink # kpt-set: ${logging-project-id}-google-workspace-data-access-sink
+  name: org-log-sink-data-access-logging-project-id # kpt-set: org-log-sink-data-access-${logging-project-id}
   namespace: logging
   annotations:
-    config.kubernetes.io/depends-on: security-log-bucket # kpt-set: logging.cnrm.cloud.google.com/namespaces/logging/LoggingLogBucket/${security-log-bucket}
+    config.kubernetes.io/depends-on: security-log-bucket # kpt-set: logging.cnrm.cloud.google.com/namespaces/logging/LoggingLogBucket/security-log-bucket
```

Comment:
I think the kpt-set can be removed and the value can become : logging.cnrm.cloud.google.com/namespaces/logging/LoggingLogBucket/security-log-bucket

Reply:
That's a good catch; I missed it when trying to mirror the changes I did in the core landing zone. Thanks. Please resolve if you're happy.